src/main/java/uk/ac/ebi/ampt2d/metadata/persistence/entities/Auditable.java

```diff
@@ -17,8 +17,27 @@
  */
 package uk.ac.ebi.ampt2d.metadata.persistence.entities;
 
-public interface BaseEntity<ID> {
+import org.springframework.data.annotation.LastModifiedDate;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
 
-    ID getId();
+import javax.persistence.EntityListeners;
+import javax.persistence.MappedSuperclass;
+import java.time.ZonedDateTime;
 
+@MappedSuperclass
+@EntityListeners(AuditingEntityListener.class)
+public abstract class Auditable<ID> {
+
+    @LastModifiedDate
+    private ZonedDateTime lastModifiedDate;
```

Comment:
Please check is there a way to display only on get request , not on post/patch request

Reply:
Fixed in 753d68e.